packages/react-components/theme-designer/src/components/ExportButton/ExportButton.tsx

```diff
@@ -10,28 +33,112 @@ const useStyles = makeStyles({
     justifyContent: 'flex-end',
     paddingRight: '20px',
   },
+  popover: {
+    width: '300px',
+  },
+  text: {
+    display: 'flex',
+    height: '31em',
+  },
 });
 
 export const ExportButton = () => {
   const styles = useStyles();
 
+  const appState = useContextSelector(AppContext, ctx => ctx.appState);
+  const { brand, lightOverrides, darkOverrides } = appState;
+
+  const [selectedValue, setSelectedValue] = React.useState<TabValue>('Code');
+
+  const onTabSelect = (event: SelectTabEvent, data: SelectTabData) => {
+    setSelectedValue(data.value);
+  };
+
+  const codeValue = dedent`
+  const brand = ${JSON.stringify(brand, null, 2)};
+
+  const lightTheme = {
+    ...createLightTheme(brand), ${getBrandValues(brand, lightOverrides, '\u00A0\u00A0')}
+  };
+
+  const darkTheme = {
+    ...createDarkTheme(brand), ${getBrandValues(brand, darkOverrides, '\u00A0\u00A0')}
+  };
+  `;
+
+  const jsonValue = dedent`
+  const lightTheme = ${JSON.stringify(
+    {
+      ...createLightTheme(brand),
+      ...lightOverrides,
+    },
+    null,
+    2,
+  )};
+
+  const darkTheme = ${JSON.stringify(
+    {
+      ...createDarkTheme(brand),
+      ...darkOverrides,
+    },
+    null,
+    2,
+  )};
+  `;
+
+  const exportedValue = () => {
```

Comment:
Nit: You could use useMemo here so you don't have to call this function every render.